pillar/environment-continuumtest-public.sls

```diff
@@ -89,7 +89,7 @@ search:
         requests_batch: 20
 
     elasticsearch:
-        servers: http://end2end--search--1.elife.internal:9200
+        servers: http://continuumtest--search.elife.internal:9200
```

Comment:
would prefer `continuumtest--search--1` though, seeing if this breaks first

Reply:
that machine doesn't have an entry for `continuumtest--search--1` in it's hosts, just `continuumtest--search`. Assuming we have `continuumtest--search--1.elife.internal` in the `elife.internal` DNS routing somewhere it should have also worked.

Reply:
Yes, `continuumtest--search--1.elife.internal` resolved through the Route53 DNS.